### DISCUSSION TITLE
Migrate wordpress website to saleor

### DISCUSSION THREAD

**Discussion by BeepLoveKarki** [saleor/saleor#7112] [Q&A]
Do we think this can be possible or is already done?

**Comment by krzysztofwolski** [comment]
It can be done. Considering that Wordpress installs are usually customized, you would need to write a custom script to migrate the data.

**Comment by KonstantinosPapadopoulos** [comment]
Take a look at this https://github.com/grll/saleor-gql-loader. The code is very well documented and it's fairly easy to customize it for your site's data.

**Comment by krzysztofwolski** [comment]
Moving to discussions.
